content/chronograf/v1.8/administration/managing-security.md

```diff
@@ -441,6 +447,24 @@ export AUTH_DURATION=1080h
 To require re-authentication every time the browser is closed, set `AUTH_DURATION` to `0`.
 This makes the cookie transient (aka "in-memory").
 
+## Configure Chronograf to authenticate with a username and password
+
+Chronograf can be configured to authenticate users by username and password ("basic authentication").
+
+{{% warn %}}
+[OAuth 2.0](#configure-chronograf-to-authenticate-with-oauth-20) is the prefered method for authentication.
+Only use basic authentication in cases where an OAuth 2.0 integration is not possible.
+{{% /warn %}}
+
+When using basic authentication, *all users have SuperAdmin status*.
+With basic authentication enabled, Chronograf authorization rules are not enforced.
+For more information, see [Cross-organization SuperAdmin status](/chronograf/v1.8/administration/managing-chronograf-users/#cross-organization-superadmin-status).
+
+Turn on HTTP basic authentication access to restrict HTTP requests to Chronograf to selected users.
```

Comment:
Is basic authentication always "HTTP" basic authentication in this context? Wondering if we should clarify this elsewhere, or simply say "Turn on basic auth..." here?

Reply:
To turn on....to selected users, open the `chronograf` CLI, and run `chronograf --htpasswd.....` (to be sure, is there something missing between chronograf and --htpasswd?)

should we clarify how to set up `HTPASSWD`?

Reply:
It looks fairly involved and requires research into more third-party tools, it seems. Let's link out for now and come back if folks ask for more?